src/components/caption/index.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+import formatCredits from "../../utils/format-credits";
+
+const COMPONENT_CLASS_NAME = "c-caption";
+
+const Caption = ({ caption, className, credit, title }) => {
+	const formattedCredit = formatCredits(credit);
+
+	return title || caption || formattedCredit ? (
+		<figcaption
+			className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+		>
+			{title ? (
+				<span dangerouslySetInnerHTML={{ __html: `${title} ` }} className="c-caption__title" />
```

Comment:
Should be perhaps use the const for this?
```suggestion
				<span dangerouslySetInnerHTML={{ __html: `${title} ` }} className=`${COMPONENT_CLASS_NAME}__title` />
```